field.go

```diff
@@ -1,6 +0,0 @@
-package main
```

Comment:
do we already have customers using the current `schema.json` format? What i was optimizing for was making the schema easier to hand write and read. This becomes an issue as db gets large. See https://github.com/segmentio/etl/blob/dq/mongo/schema.yml